src/Collections/Sequences/Seq.dfy

```diff
@@ -397,7 +401,8 @@ module Seq {
     requires 0 < |a| && 0 < |b|
     ensures Min(a+b) <= Min(a)
     ensures Min(a+b) <= Min(b)
-    ensures forall i {:trigger i in a + b} :: i in a + b ==> Min(a + b) <= i
+    /* Dafny selected trigger: {i in a + b} */
+    ensures forall i {:trigger i in a + b} :: i in a + b ==> Min(a + b) <= i 
```

Comment:
Seems like Dafny agrees with the manual trigger here?

Reply:
Fixed.